web/source/osk/emptyView.ts

```diff
@@ -0,0 +1,17 @@
+/// <reference path="keyboardView.interface.ts" />
+
+namespace com.keyman.osk {
+  export class EmptyView implements KeyboardView {
+    readonly element: HTMLDivElement;
```

Comment:
I'm _fairly_ certain KMW currently keeps the OSK hidden whenever this is the active "inner frame" type.  But... it _does_ bother to construct it, so it gets its own proper class.